server/polar/kit/services.py

```diff
@@ -160,12 +160,13 @@ async def update(
         update_schema: UpdateSchemaType,
         include: set[str] | None = None,
         exclude: set[str] | None = None,
+        exclude_unset: bool = False,
         autocommit: bool = True,
     ) -> ModelType:
         return await source.update(
             session,
-            include=include,
-            exclude=exclude,
```

Comment:
This was the only usage of these parameters in `ActiveRecordMixin.update()`, it might be worth deleting them there as well.